common/nodes/database-schema.xml

```diff
@@ -220,6 +208,19 @@ CREATE TABLE rolls (
 	INDEX (Name)
 );
 
+<!-- Tags -->
+DROP TABLE IF EXISTS tags;
+CREATE TABLE tags (
+	scope		ENUM('box', 'cart', 'network', 'pallet') NOT NULL,
+	tag		VARCHAR(128) NOT NULL,
+	value		TEXT,
+	scopeid		INT NOT NULL,
+	INDEX (scope),
+	INDEX (tag),
+	INDEX (scopeid),
```

Comment:
You don't need this line creating an index on `scopeid` because you get an index with the FOREIGN KEY on the next line.